examples/ArduinoIoTCloud-DeferredOTA/thingProperties.h

```diff
@@ -34,4 +35,9 @@ void initProperties() {
   LoRaConnectionHandler ArduinoIoTPreferredConnection(SECRET_APP_EUI, SECRET_APP_KEY, _lora_band::EU868, NULL, _lora_class::CLASS_A);
 #elif defined(BOARD_HAS_NB)
   NBConnectionHandler ArduinoIoTPreferredConnection(SECRET_PIN, SECRET_APN, SECRET_LOGIN, SECRET_PASS);
+#elif defined(BOARD_HAS_ETHERNET)
+  /* DHCP mode */
+  //EthernetConnectionHandler ArduinoIoTPreferredConnection;
+  /* Manual mode. It will fallback in DHCP mode if SECRET_OPTIONAL_IP is invalid or equal to "0.0.0.0" */
```

Comment:
Okay, the parameter is optional when using DHCP. But maybe a better naming would be `SECRET_MANUAL_IP`. Also its not really a secret, just a configuration item. Maybe `ETHERNET_MANUAL_CONFIG_IP`?

Reply:
Thank you for your [clarification](https://github.com/arduino-libraries/ArduinoIoTCloud/pull/342#issuecomment-1306877015). I thought so. In this case there's not much you do I suppose :shrug:.